alf/layers.py

```diff
@@ -126,6 +126,11 @@ def __init__(self,
             bias_init_value (float): a constant
         """
         super(FC, self).__init__()
+        # get the argument list with vals
+        self._kwargs = locals()
```

Comment:
suggest to make a copy of locals(). Otherwise, future addition of local variable will change self._kwargs

Reply:
updated